docs/adrs/0077-permanently-store-origin-duty-gbloc.md

```diff
@@ -0,0 +1,114 @@
+  * How disruptive will this work be on other teams?
+* Flexibility
+  * Will this solution support required future changes?
+  * Will it be easy and intuitive to handle GBLOC consolidations and additions?
+* Consistency
+  * Does this change follow best practices?
+  * Does it introduce any anti-patterns?
+
+
+## Considered Alternatives
+
+> **bold denotes chosen**
+
+* *Do nothing*
+* *Surface Origin GBLOC data in History & Audit Log*
+* **Add a column to the orders table**
+* *Store the Origin Duty Location GBLOC in a join table*
+
+## Decision Outcome
+
+<!-- * Chosen Alternative:  -->
+* Chosen Alternative: *Add a column to the orders table*
+* Positive Outcomes: 
+  * Eliminate relying on a view for frequently accessed data
+  * Provides a direct link from an orders record to its GBLOC, which would flow to warehouse
+* Consequences: 
+  * Event triggers will need to be updated for history and audit logging purposes
+* Other considerations:
+  * Data backfill will need to be addressed
+
+## Pros and Cons of the Alternatives
+
+### *Do nothing*
+Continue using the views as we currently are, and address existing bugs and edge cases as needed.
+
+* `+` *There's no work to be done so teams can focus on other work.*
+* `+` *No data backfill required*
+* `-` *GBLOC data remains dynamic and no view data is warehoused*
+* `-` If Postal code to GBLOC relationship changes in the future, there is no historical record in MilMove
+of what the GBLOC was at the time of the move.
+
+### *Surface Origin Duty Location GBLOC data in History & Audit Log*
+Continue using the views as we currently do, address existing bugs and edge cases as needed, but 
+visually display GBLOC data in the history and audit log alongside the duty location data.
+
+* `+` *No significant changes means lower level of impact*
+  * `+` Could be a "quick win"
+* `+` *Client can see visual reference to a move's GBLOC*
+* `+` *No data backfill required*
+* `-` *Adds another join to the move history fetcher SQL query*
+* `-` *GBLOC data remains dynamic and no view data is warehoused*
+* `-` If Postal code to GBLOC relationship changes in the future, there is no historical record in MilMove
+of what the GBLOC was at the time of the move.
+
+### *Add a column to the orders table*
+Reintroduce the `gbloc` column to the `orders` table. This column was included when the table was originally created,
+but later dropped in favor of using the view to fetch the origin duty location GBLOC.
+
+* `+` *Single column to add*
+* `+` *Reduces the number of joins when querying for move task orders.*
+* `+` *Eliminates the need for the `origin_duty_location_to_gbloc` view and associated models, potential positive performance impact.*
+* `-` *Backfill data will need to be generated*
+  * `+` We can leverage the SQL query currently used for the `origin_duty_location_to_gbloc` view to backfill data
+* `-` *Will require DB trigger update/maintenence to be incorporated into the history & audit log*
```

Comment:
Update the spelling of `maintenence` throughout the ADR.
```suggestion
* `-` *Will require DB trigger update/maintenance to be incorporated into the history & audit log*
```

Reply:
This is an embarrassing one. Thanks.